src/scripts/equation_controller.js

```diff
@@ -0,0 +1,54 @@
+import { Controller } from 'stimulus-repo/packages/stimulus'
+
+export class EquationController extends Controller {
+  static targets = ['problem', 'answer']
+  static parent = 'quiz'
+
+  randomNumber() {
+    return Math.floor(Math.random() * 10) + 1
+  }
+
+  generateProblem() {
+    this.a = this.randomNumber()
+    this.b = this.randomNumber()
+    this.problemTarget.textContent = `${this.a} * ${this.b}`
+  }
+
+  connect() {
+    this.generateProblem()
+  }
+
+  correct() {
+    const element = this.answerTarget
+    const answer = parseInt(element.value)
+    return answer === this.a * this.b
+  }
+
+  wrong() {
+    this.problemTarget.classList.replace('is-static', 'is-danger')
+    this.answerTarget.classList.add('is-danger')
```

Comment:
This is the code that sets the equation to red when it is wrong. I showed this to a friend and he reminded me that color blind people have trouble distinguishing between red and green. So I added an outline to the answer as well, so there is another visual clue that it is wrong.